Specs/DateTime/Japanese/TimeExtractor.json

```diff
@@ -721,12 +665,19 @@
   },
   {
     "Input": "今日の１時",
```

Comment:
Please make sure we have the full case in datetime.

Reply:
added